head/tests/test_presence_sensing_driver.cpp

```diff
@@ -18,4 +23,41 @@ SCENARIO("get readings called on presence sensing driver") {
             }
         }
     }
+}
+
+SCENARIO("get_tool called on presence sensing driver") {
+    GIVEN("PresenceSensingDriver instance") {
+        auto raw_readings =
```

Comment:
This test only tests one case. It doesn't test for the other tools or for there being no tools.